### PR TITLE
filledcurves with above/below threshold and fillcolor implemented.

### PR DESCRIPTION
**Pull request by vikasnkumar**
example added as well. tested and it works.

### CODE REVIEW

File: examples/demo.pl [hunk 1]
```diff
@@ -85,6 +84,15 @@ sub prompt {
...
 
+## filled curve
+$w->plot(
+    { with => 'filledcurves', fc => 'red', below => 'y=0' },
+    -$x, (-$x)**3,
+    { with => 'filledcurves', fillcolor => 'green', above => 'y=0' },
+    $x, ($x)**3
+);
+prompt("Area curve with filled colors");
+
```
Comment (mohawk2) [inline examples/demo.pl:95]: Can you drop this blank line? Thanks.

File: lib/PDL/Graphics/Gnuplot.pm [hunk 2]
```diff
@@ -5266,7 +5300,8 @@ our $cOptionsAbbrevs = _gen_abbrev_list(keys %$cOptionsTable);
 	lc => ["linecolor"],
 	pt => ["pointtype"],
 	ps => ["pointsize"],
-	fs => ["fillstyle"]
+	fs => ["fillstyle"],
+	fc => ["fillcolor"]
```
Comment (mohawk2) [inline lib/PDL/Graphics/Gnuplot.pm:5304]: Can you please add a `,` on the end, so one could add another line here without spurious diffs, given Perl allows that?

**Review by mohawk2 — CHANGES_REQUESTED** [review]

**Comment by vikasnkumar** [comment]
@mohawk2 any updates here ? is this ready for release ?

**Review by mohawk2 — APPROVED** [review]

**Comment by mohawk2** [comment]
Thanks! Just released as 2.032.

**Pull request merged**